x/ibc/07-tendermint/update.go

```diff
@@ -85,8 +85,11 @@ func checkValidity(
 	}
 
 	// Verify next header with the last header's validatorset as trusted validatorset
-	err := lite.Verify(clientState.GetChainID(), &clientState.LastHeader.SignedHeader, clientState.LastHeader.ValidatorSet,
-		&header.SignedHeader, header.ValidatorSet, clientState.TrustingPeriod, currentTimestamp, lite.DefaultTrustLevel)
+	err := lite.Verify(
```

Comment:
`Verify` now requires a `maxClockDrift time.Duration`. What value should this be?

@cwgoes @marbar3778 @AdityaSripal

Reply:
It should be a parameter on client creation.

Reply:
Yes please!

Reply:
I don't really have much context here. Can someone with more context please make the necessary changes on this branch?

Reply:
On the light client the default is 10 seconds

Reply:
Although it is an option that can be set by the user when the lc starts

Reply:
Ok, is that an exposed `const` we can directly use or should I just hard-code this to 10s?

Reply:
it's not exposed so just hard code it unless you want to make it a configurable parameter

Reply:
No, I mean make it configurable for IBC light clients.

Reply:
You mean update `ClientState` and `MsgCreateClient` @cwgoes ?

Reply:
Yes, precisely.

Reply:
Added; Do envision any necessary stateless validation or bounds an allowed max clock drift?

Reply:
Thanks! IBC users have to inspect client state anyways, so no bounds needed there, but there may be bounds needed to avoid panics from the Tendermint side - cc @cmwaters

Reply:
No bounds that I know of, it's just the higher the number the more it makes the time related verification obsolete